R/round.r

```diff
@@ -159,10 +161,10 @@ parse_unit_spec <- function(unitspec) {
     mult <- as.numeric(parts[[1]])
     unit <- parts[[2]]
   }
-  
+
```

Comment:
> Unit test fail. 

Ok. It's the doc warning. We can leave with this. If it's not difficult, could you please re-roxygenize with the dev version of roxygen2 or adjust the Rd manually.